fs2/prediction_writing_callback.py

```diff
@@ -402,13 +400,18 @@ def on_predict_batch_end(  # pyright: ignore [reportIncompatibleMethodOverride]
             wavs,
             outputs["tgt_lens"],
         ):
-            write(
-                self._get_filename(
-                    basename=basename,
-                    speaker=speaker,
-                    language=language,
+            torchaudio.save(
```

Comment:
Is the change of audio writer function related to this PR, or just an unrelated improvement? I assume you've tested and you can confirm this works well?

Reply:
yes just a consolidation/refactor since we use torchaudio everywhere else. tested and works for me.